lib/netsuite/support/actions.rb

```diff
@@ -28,8 +28,6 @@ def action(name)
             self.send(:include, NetSuite::Actions::GetSelectValue::Support)
           when :search
             self.send(:include, NetSuite::Actions::Search::Support)
-          when :search_more_with_id
```

Comment:
This is a bug in the gem - broken master.  There is a PR in the main repo
to do the same.  Someone deleted the SearchMoreWithId action but didn't
clean everything up properly.

On Wed, Feb 25, 2015 at 8:05 PM, Jerome Cornet <notifications@github.com>
wrote:

> In lib/netsuite/support/actions.rb
> <https://github.com/prburke/netsuite/pull/2#discussion_r25396634>:
>
> > @@ -28,8 +28,6 @@ def action(name)
> >              self.send(:include, NetSuite::Actions::GetSelectValue::Support)
> >            when :search
> >              self.send(:include, NetSuite::Actions::Search::Support)
> > -          when :search_more_with_id
>
> why did you remove it here and above ?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/prburke/netsuite/pull/2/files#r25396634>.
>